apps/aechannel/src/aesc_fsm.erl

```diff
@@ -999,12 +1054,14 @@ wdraw_half_signed(cast, {Req,_} = Msg, D) when ?UPDATE_CAST(Req) ->
     %% arrived.
     lager:debug("race detected: ~p", [Msg]),
     handle_update_conflict(Req, D);
-wdraw_half_signed(cast, {?WDRAW_SIGNED, Msg}, #data{latest = Latest} = D) ->
+wdraw_half_signed(cast, {?WDRAW_SIGNED, Msg}
+                  , #data{op = #op_ack{tag = withdraw_tx} = Op} = D) ->
```

Comment:
I don't know ... it feels more natural to have that comma on the preceding line.